src/backend/gl/src/lib.rs

```diff
@@ -486,6 +497,36 @@ impl CommandQueue {
                 self.state.index_buffer = Some(0);
             }
         }
+
+        // Reset viewports
+        if self.state.num_viewports == 1 {
+            unsafe { gl.Viewport(0, 0, 0, 0) };
+            unsafe { gl.DepthRange(0.0, 1.0) };
+        } else if self.state.num_viewports > 1 {
+            // 16 viewports is a common limit set in drivers.
+            let viewports: SmallVec<[[f32; 4]; 16]> =
```

Comment:
16 is not the upper bound, even though from gpu info database, it seems to be the one reported by most drivers atm. So in the theory/future we could have more than 16.